content/doc/developer/cli/writing-cli-commands.adoc

```diff
@@ -53,11 +53,11 @@ It doesn't matter where the resolver method is defined, or how it's named.
 
 Both resolver methods and CLI methods can have any number of link:https://args4j.kohsuke.org[args4j] annotations, which causes the parameters and arguments to be injected upon a method invocation. 
 All the other unannotated parameters receive null. 
-Combined with the stapler method binding, this enables you to make your method invokable from both CLI and HTTP.
+Combined with the stapler method binding, this enables you to make your method invocable from both CLI and HTTP.
```

Comment:
Questionable given https://en.wiktionary.org/wiki/invocable

Reply:
its probably a us-en dictionary, I can add an override, but as a uk-en/ca-en I think its fine

Reply:
Several other terms switched from alternative to preferred (per Wiktionary), so I'm a bit confused.

Reply:
> us-en dictionary, I can add an override, but as a uk-en/ca-en I think its fine

Per https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc the site is American English.

Reply:
Interesting, while `invokable` is the preferred spelling in Wiktionary, Merriam-Webster doesn't even have it, instead only https://www.merriam-webster.com/dictionary/invocable

In terms of BE/AE, that's not how it works -- the tool maintains a list of misspellings and suggested corrections: https://github.com/crate-ci/typos/blob/master/docs/design.md

So I think we're good here.